src/screens/DiscoveryScreenV2/elements/DiscoveryTab.js

```diff
@@ -1,44 +1,64 @@
 import * as React from 'react';
 import {Dimensions, Keyboard, Pressable, ScrollView, StyleSheet, Text, View} from 'react-native';
 
+import {useRoute} from '@react-navigation/core';
+import PropTypes from 'prop-types';
 import {colors} from '../../../utils/colors';
-import {fonts, normalizeFontSize} from '../../../utils/fonts';
+import {normalizeFontSize} from '../../../utils/fonts';
+import {setNavbarTitle} from '../../../context/actions/setMyProfileAction';
+import {Context} from '../../../context';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been modified to include new dependencies. Ensure that these new dependencies are correctly installed and used in the code.
```diff
- import {colors} from '../../../utils/colors';
- import {fonts, normalizeFontSize} from '../../../utils/fonts';
+ import {useRoute} from '@react-navigation/core';
+ import PropTypes from 'prop-types';
+ import {colors} from '../../../utils/colors';
+ import {normalizeFontSize} from '../../../utils/fonts';
+ import {setNavbarTitle} from '../../../context/actions/setMyProfileAction';
+ import {Context} from '../../../context';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->